easybuild/easyconfigs/s/SPLASH/SPLASH-2.8.0-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'SPLASH'
+version = '2.8.0'
+
+homepage = "http://users.monash.edu.au/~dprice/splash/index.html"
+description = """SPLASH is a free and open source visualisation tool 
+for Smoothed Particle Hydrodynamics (SPH) simulations."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['http://users.monash.edu.au/~dprice/splash/releases/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['de06b159846bd20717a55ec7cc9420e0fd86ab38dc66dc74b1f61b77b4d24fce']  # splash-2.8.0.tar.gz
+
+dependencies = [
+    ('Giza', '1.1.0'),
+]
+
+skipsteps = ['configure']
+buildopts = 'SYSTEM=gfortran X11_DIR=$EBROOTX11 '
```

Comment:
@dquigley-warwick Can we avoid hardcoding the `gfortran` here which is closely tied to the `foss` toolchain?

Can we use `$F90` instead for example, which would be `gfortran` when `foss` toolchain is used, `ifort` when `intel` toolchain is used, etc.?